ui/src/pages/backoffice/EItem/EItemDetails/components/EItemFiles/index.js

```diff
@@ -0,0 +1,27 @@
+import { connect } from 'react-redux';
+
+import EItemFilesComponent from './EItemFiles';
+import { uploadFile, deleteFile } from '../../state/actions';
+import { addNotification } from '@components/Notifications';
+
+const mapStateToProps = state => ({
+  eitemDetails: state.eitemDetails.data,
+  files: state.eitemDetails.files,
+  error: state.eitemDetails.error,
+  isLoading: state.eitemDetails.isLoading,
+  isFilesLoading: state.eitemDetails.isFilesLoading,
+  hasError: state.eitemDetails.hasError,
```

Comment:
`...state.eitemDetails` this could also work

Reply:
True! But I noticed I didn't actually need all of them so I added only the ones I needed.